.env

```diff
@@ -0,0 +1,5 @@
+COMPOSE_PROJECT_NAME=temporal
+CASSANDRA_VERSION=3.11
+ELASTICSEARCH_VERSION=8.0.0
+MYSQL_VERSION=8
+POSTGRESQL_VERSION=13
```

Comment:
I might have missed it; what step uses this?

Reply:
These are default values used by docker-compose